packages/esm-forms-dashboard-app/src/config-schema.ts

```diff
@@ -0,0 +1,81 @@
+import { validator, Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  formsSectionsConfig: {
+    _type: Type.Array,
+    _elements: {
+      name: {
+        _type: Type.String,
+        _description: 'Name of the section.',
+      },
+      labelCode: {
+        _type: Type.String,
+        _description: 'Label to be used to translate the section.',
+      },
+      forms: {
+        _type: Type.Array,
+        _description: 'Configuration of the forms.',
+        _elements: {
+          name: {
+            _type: Type.String,
+            _description: 'Name of form.',
+          },
+          encounterTypeUuid: {
+            _type: Type.String,
+            _description: 'Uuid fo the encounter type.',
+          },
+        },
+      },
+    },
+    _default: undefined,
+  },
+  showHtmlFormEntryForms: {
+    _type: Type.Boolean,
+    _default: true,
+    _description: 'Whether HTML Form Entry forms should be included in lists of forms.',
+  },
+  showRecommendedFormsTab: {
+    _type: Type.Boolean,
+    _description: 'Whether to display recommended forms tab. Requires AMPATH ETL.',
+    _default: false,
+  },
+  showConfigurableForms: {
+    _type: Type.Boolean,
+    _description: 'Whether to use customURL to fetch forms, works with `customFormsUrl`',
+    _default: false,
+  },
+  customFormsUrl: {
+    _type: Type.String,
+    _description: 'Custom forms endpoint to fetch forms using a custom url need `showConfigurableForms`',
+    _default: '',
+  },
```

Comment:
Can't we just combine these into one setting? I.e., if you have a `customFormsUrl` then we use that and if not we use the default?